Elevel/Elevel.Api/Controllers/ApplicationUserController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Elevel.Application.Features.ApplicationUserFeatures;
+using Elevel.Domain.Enums;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace Elevel.Api.Controllers
+{
+    [Authorize(Roles = nameof(UserRole.HumanResourceManager))]
+    [Route("api/[controller]")]
+    public class ApplicationUserController : BaseApiController
+    {
+        [Authorize, HttpGet("")]
```

Comment:
if you don't change auth level (role or allowAnonymous) you can just omit Authorize, because you've already have it on Controller.